spec/features/shelters/show_spec.rb

```diff
@@ -6,6 +6,47 @@
     @shelter_2 = Shelter.create(name: "Alex's Shelter", address: "345 Georgia Street", city: "Denver", state: "CO", zip: "80223")
     @shelter_3 = Shelter.create(name: "Aaron's Shelter", address: "678 California Street", city: "Denver", state: "CO", zip: "80234")
 
+    @pet_1 = @shelter_1.pets.create!(
+        image: 'https://s3.amazonaws.com/cdn-origin-etr.akc.org/wp-content/uploads/2017/11/12234558/Chinook-On-White-03.jpg',
+        name: "Penny",
+        age: 2,
+        sex: "F",
+        description: "Nice dog"
+        )
+
+    @pet_2 = @shelter_1.pets.create!(
+        image: 'https://s3.amazonaws.com/cdn-origin-etr.akc.org/wp-content/uploads/2017/11/12234558/Chinook-On-White-03.jpg',
+        name: "Rufus",
+        age: 2,
+        sex: "M",
+        description: "Big dog"
+        )
+
+    @owner_info1 = {  :name => 'Alex',
+                    :address => '123 Maine Ave',
+                    :city => 'Paris',
+                    :state => 'CO',
+                    :zip => '80210',
+                    :phone_number => '2077020720',
+                    :description => 'I will be a good pet owner, trust me.'}
+
+    @owner_info2 = { :name => 'Mike',
+                    :address => '476 Market street',
+                    :city => 'Denver',
+                    :state => 'CO',
+                    :zip => '80210',
+                    :phone_number => '4737020720',
+                    :description => "I loved pets, I'll do my best"}
+
+    @application1 = @pet_1.applications.create(@owner_info1)
+    @application2 = @pet_2.applications.create(@owner_info2)
+
+    @review_1 = @shelter_1.reviews.create(title: "Good shelter", rating: 5, content: "It was a great shelter", image: "https://i.pinimg.com/originals/d1/21/f6/d121f64f59f7599486e3762fcd2f1475.jpg")
+    @review_2 = @shelter_1.reviews.create(title: "Okay shelter", rating: 3, content: "It was an okay shelter", image: "https://i.pinimg.com/originals/d1/21/f6/d121f64f59f7599486e3762fcd2f1475.jpg")
```

Comment:
we need to check that the correct number is also returned when some reviews have been added to a different `shelter`